app/src/main/java/io/github/droidkaigi/confsched2018/presentation/MainActivity.kt

```diff
@@ -70,6 +75,25 @@ class MainActivity : BaseActivity(), HasSupportFragmentInjector {
         binding.bottomNavigation.setOnNavigationItemReselectedListener { }
     }
 
+    override fun onRestoreInstanceState(savedInstanceState: Bundle?) {
+        super.onRestoreInstanceState(savedInstanceState)
+        if (savedInstanceState != null) {
+            val navigationItem: BottomNavigationItem =
+                    BottomNavigationItem.forId(binding.bottomNavigation.selectedItemId)
+            Timber.d("menu: %s", navigationItem)
+            supportActionBar?.apply {
```

Comment:
Can you share it with similar sentences above? 
With a name such as `updateToolbar(bottomNavigationItem : BottomNavigationItem)`